src/pip/_internal/network/lazy_wheel.py

```diff
@@ -1,66 +1,122 @@
 """Lazy ZIP over HTTP"""
 
-__all__ = ["HTTPRangeRequestUnsupported", "dist_from_wheel_url"]
+from __future__ import annotations
```

Comment:
I seem to recall there was a decision to not use this until… I don’t even remember the context. Whatever, I won’t object unless someone recalls it.

Reply:
This is super easy to remove if needed!

Reply:
We were 3.6+ when this was last discussed, IIRC. And, yea, let's keep this unless someone voices major concerns.